docs/pages/repo/docs/core-concepts/caching.mdx

```diff
@@ -130,7 +130,8 @@ You can also disable caching on specific tasks by setting the [`pipeline.<task>.
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "dev": {
-      "cache": false
+      "cache": false,
```

Comment:
Does `persistent: true` automatically set `cache` to `false`?

If not, then it probably should?

And if it does, we should remove these `cache: false` keys and document that you don't need to add them.

Reply:
It does not, at the moment. Might be a good consolidation down the road, but seems like an overreach when we introduce this. `cache:false` also means ignoring `outputs`, which has some baggage also.